core/src/main/java/be/isach/ultracosmetics/cosmetics/particleeffects/ParticleEffectEnderAura.java

```diff
@@ -19,6 +19,6 @@ public ParticleEffectEnderAura(UltraPlayer owner, UltraCosmetics ultraCosmetics)
 
 	@Override
 	public void onUpdate() {
-		getPlayer().getWorld().playEffect(getPlayer().getLocation().add(0, 1, 0), Effect.ENDER_SIGNAL, 0);
+                getPlayer().getWorld().spawnParticle(Particle.PORTAL, getPlayer().getLocation().add(0, 1, 0), 3, 0.2D, 0.01D, 0.2D);

```

Comment:
Only 1.13 compatible. Use the [UtilParticles](https://github.com/iSach/UltraCosmetics/blob/master/core/src/main/java/be/isach/ultracosmetics/util/UtilParticles.java) class.
```suggestion
                Particles.PORTAL.display(0.2F, 0.01F, 0.02F, 0, 3, getPlayer().getLocation().add(0, 1, 0), 128);
```